core/src/main/java/com/linecorp/armeria/client/circuitbreaker/CircuitBreakerStrategy.java

```diff
@@ -65,6 +70,27 @@
      * {@link CircuitBreaker#onSuccess()} is called so that the {@link CircuitBreaker} increases its success
      * count and use it to make a decision to close or open the switch. If {@code false} is returned, it works
      * the other way around. If {@code null} is returned, the {@link CircuitBreaker} ignores it.
+     *
+     * @param ctx the {@link ClientRequestContext} of this request
+     * @param response the {@link Response} from the server.
+     *                 If {{@link #needsResponseStreaming(ClientRequestContext)}} returns {@code false} and
+     *                 the type of {@link Response} is {@link HttpResponse}, this will be {@code null}.
+     *                 This will be the actual {@link Response} otherwise.
```

Comment:
How about:

```
{@code null} if {@link #needsResponseStreaming(...)} ....
The actual {@link Response} otherwise.
```

Please also note the double curly braces in `{{@link #needsResponseStreaming...}}`